src/main/java/teammates/ui/controller/InstructorCourseEnrollSaveAction.java

```diff
@@ -43,7 +44,7 @@ public ActionResult execute() throws EntityDoesNotExistException {
                                                                     courseId, students, hasSection, studentsInfo);
             
             statusToAdmin = "Students Enrolled in Course <span class=\"bold\">[" 
-                            + courseId + "]:</span><br>" + (studentsInfo).replace("\n", "<br>");
+                            + courseId + "]:</span><br>" + Sanitizer.sanitizeForHtml((studentsInfo)).replace("\n", "<br>");
```

Comment:
In line 53 below, we have a similar statement. Please sanitize it as well. But before that, let's introduce another String variable named `sanitizedStudentsInfo` then use it for both places.